app/feature/feature-edit-coinsured/src/main/kotlin/com/hedvig/android/feature/editcoinsured/data/FetchCoInsuredPersonalInformationUseCase.kt

```diff
@@ -49,5 +49,10 @@ private fun convertSsnToBirthDateOrNull(ssn: String): LocalDate? {
   val stringDate = ssn.substring(6, 8).toIntOrNull()
   if (stringYear == null || stringMonth == null || stringDate == null) return null
   if (stringMonth !in 1..12) return null
-  return LocalDate(year = stringYear, month = Month(stringMonth), dayOfMonth = stringDate)
+  val result = try {
```

Comment:
Perhaps along with this change we can look if we can also get rid of that `formatShortSsn` function as discussed here https://hedviginsurance.slack.com/archives/C03HT2JRDPG/p1718007153791809 .
If we do not use it anywhere else of course and what I am saying there makes sense.